.github/workflows/build-test-package.yml

```diff
@@ -274,22 +278,29 @@ jobs:
         export MANYLINUX_VERSION=`(echo ${MANYLINUX_PLATFORM} | cut -d '-' -f 1)`
         export TARGET_ARCH=`(echo ${MANYLINUX_PLATFORM} | cut -d '-' -f 2)`
         export ITK_DIR_NAME=${{ matrix.itk-package }}${MANYLINUX_VERSION}_${TARGET_ARCH}
+
         cat > git-configure-build-sem.sh << EOF
         #!/bin/sh
-        echo "/ITKPythonPackage"
-        ls -la /ITKPythonPackage
         ln -s -f -T /ITKPythonPackage/${ITK_DIR_NAME} /work/${ITK_DIR_NAME}
         ln -s -f -T /ITKPythonPackage/ITK-source /work/ITK-source
         ln -s -f -T /ITKPythonPackage/oneTBB-prefix /work/oneTBB-prefix
-        echo "/work"
-        ls -la /work
         EOF
         chmod +x ./git-configure-build-sem.sh
-        echo "/tmp"
-        ls -la /tmp
-        /tmp/dockcross-manylinux-x64 -a "-v $PWD/ITKPythonPackage:/ITKPythonPackage" ./git-configure-build-sem.sh
 
-    - name: 'Git-Configure-Build VTK'
+        source "./ITKPythonPackage/scripts/dockcross-manylinux-set-vars.sh"
+        docker run -v $(pwd):/work/ -v $PWD/ITKPythonPackage:/ITKPythonPackage --rm ${CONTAINER_SOURCE} ./git-configure-build-sem.sh
```

Comment:
Note that this line is currently attempting to run an ARM container on an x64 host, which results in an error:

> WARNING: The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64/v3) and no specific platform was requested
/usr/local/bin/manylinux-entrypoint: line 8: /git-configure-build-sem.sh: No such file or directory

To address this you will need to install ARM emulation tools. See https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/master/scripts/dockcross-manylinux-build-module-wheels.sh#L78

Cross-platform emulation tools: [binfmt](https://github.com/tonistiigi/binfmt#binfmt)

Note that cross-platform emulation is the primary reason that ITK `aarch64` module wheels build much slower than `x64` wheels on GitHub Actions. This may be improved in the future with the `dockcross/manylinux2014-aarch64` image. If the build time is prohibitive, there are also other at-cost options we could explore in the near term for building on custom ARM runners.

Reply:
Okay, it looks like this is the actual failure:
> /usr/local/bin/manylinux-entrypoint: line 8: /git-configure-build-sem.sh: No such file or directory

Possibly a container path issue. Try `/work/git-configure-build-sem.sh`, i.e.

```sh
docker run -v $(pwd):/work/ -v $PWD/ITKPythonPackage:/ITKPythonPackage --rm ${CONTAINER_SOURCE} /work/git-configure-build-sem.sh
```

If you continue to see an error here and you have access to a local Linux setup, you might try running the docker container locally to inspect the container filesystem and see how directories are being mounted.